tests/Dfc.CourseDirectory.WebV2.Tests/FeatureTests/DataManagement/Courses/UploadTests.cs

```diff
@@ -359,6 +359,11 @@ public async Task Post_FileWithLarsErrors_RendersExpectedResult()
 
             var doc = await response.GetDocument();
             doc.AssertHasError("File", "The file contains errors and could not be uploaded");
+
+            var missingLars = doc.GetAllElementsByTestId("MissingLars").Select(e => e.TextContent.Trim());
```

Comment:
Should these be removed if the variables aren't used?

Reply:
Yes you are right. I will remove them and commit the changes